internal/proto/v1/hook.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
```

Comment:
Can this file be located in the  cmd/tusd/cli/hooks/ folder where the other hook related functionality is located?